alf/trainers/on_policy_trainer.py

```diff
@@ -150,6 +151,9 @@ def train_():
                     tf.summary.text('commandline', ' '.join(sys.argv))
 
         checkpointer.save(global_step=global_step.numpy())
+        env.close()
```

Comment:
No such method for TFPyEnvironment created from create_environment

Reply:
Ok, I’ll fix it